docs/chapters/projekt/analiza_i_specyfikacja_wymagan.tex

```diff
@@ -1,34 +1,74 @@
-pomogą mu we wzmacnianiu swoich postaci.
-
-Dodatkowo gra musi udostępniać podstawową funkcjonalność swojego gatunku, czyli sterowanych przez sztuczną inteligencję
-przeciwników. Powinni oni wykonywać ataki typowe dla realiów epoki oraz potrafić poprawnie określić swój cel. Co więcej,
-powinni oni inicjować walkę z graczem, gdy go napotkają. Z tego powodu konieczne będzie utworzenie mechanizmu walki, który
-obsłuży zachowanie postaci biorących w udział w starciu, jego przebieg oraz umożliwi użytkownikowi wykonywanie akcji
-swoją postacią.
-
-Konieczne będzie utworzenie interfejsu graficznego, który w czytelny sposób wyświetli graczowi aktualny stan gry, pomoże
-mu w nawigacji i poznawaniu świata oraz przedstawi możliwe do podjęcia akcje. Najważniejszymi jego cechami będą prostota
-i intuicyjność, aby nie przytłoczyć użytkownika ogromem i skomplikowaniem informacji. Jego celem ma być pogłębienie immersji
-i przedstawienie świata z punktu widzenia granej postaci. Niezbędne będą informacje o posiadanych surowcach, czasie oraz
-położeniu. Interfejs użytkownika musi też jednak dostosowywać się do sytuacji i zmieniać się tak, aby gracz miał przedstawione
-możliwe opcje podczas dowodzenia walką, budowania budynków oraz prowadzenia dialogów. Kluczowe w przedstawieniu informacji
-będzie stylistyczne wpasowanie się w realia gry, aby umożliwić graczowi wczucie się w epokę, w której jest osadzona fabuła.
+W niniejszej sekcji przedstawiono specyfikę wymagań funkcjonalnych, pozafunkcjonalnych oraz tych, wynikających z
+głównych założeń projektu. Dodatkowo zawiera on diagramy przypadków użycia, maszyny stanów oraz klas prototypowej gry.
+
+\subsection{Specyfika wymagań wynikających z założeń projektu}
+Z punktu widzenia projektu kluczowe jest jak najdokładniejsze oddanie realiów historycznych przy jednoczesnym
+uwzględnieniu jakości rozgrywki gracza oraz cech charakterystycznych dla gier typu RTS. Z założeń wynika, że fabuła
+gry powinna zostać osadzona w czasach sprzed wielkich odkryć geograficznych. Na tej podstawie zostały zdefniowane
+dodatkowe wymagania, które powinien spełniać prototyp:
+\begin{itemize}
+  \item Sposób nawigacji powinien jak najdokładniej odpowiadać temu stosowanemu w wybranej epoce
+  \item Postacie w grze powinny stylistycznie pasować do realiów historycznych
+  \item Postacie powinny posługiwać się słownictwem adekwatnym do czasów, w których osadzona jest gra
+  \item Postacie powinny jak najlepiej oddawać światopogląd w danych czasach
+  \item Oręż stosowany w grze powinien odpowiadać realiom historycznym
+  \item Budowle w grze powinny stylistycznie odpowiadać wybranej epoce
+  \item Sposób komunikacji z postaciami powinien imitować ten stosowany w danych czasach
+\end{itemize}
+
+\subsection{Wymagania funkcjonalne}\label{ss:fun}
+W liście \ref{list:fun} zostały wymienione wymagania funkcjonalne, które powinien spełniać prototyp gry.
```

Comment:
może bez odwołania do numeru listy? w pdf'ie nie jest ona podpisana, więc nie wiem czy to nie wprowadza więcej zamieszania?